Dockerfile

```diff
@@ -15,8 +15,7 @@ VOLUME /data
 
 USER gkh
 
-RUN helm init --client-only && \
-    helm plugin install https://github.com/futuresimple/helm-secrets.git || \
+RUN helm plugin install https://github.com/futuresimple/helm-secrets.git || \
```

Comment:
could you please add the changes requested in: https://github.com/kiwigrid/gcloud-kubectl-helm/commit/cba4a65e284ef680aaba79d3d54f94f765b6bfc4

Reply:
Yes